packages/prometheus/prometheus.spec

```diff
@@ -0,0 +1,59 @@
+Name:     prometheus
```

Comment:
This is a new file.
Below I can see: packages/prometheus-2.20.1/prometheus.spec → packages/prometheus/prometheus.spec.bkp 

I guess you don't want to commit the backup file to the git project.

Reply:
indeed. removed.